### PR TITLE
Compose File Fixes

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Small fixes for the docker-compose file

### CODE REVIEW

File: server/api/index.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ const restrict = (req, res, next) => {
 // Send back a simple JSON error with an HTTP status code
 const errorHandler = (err, req, res, next) => {
   if (err) {
+    console.error(err.stack)
```
Comment (delventhalz) [inline server/api/index.js:135]: So every time there is an error when a user accesses the API (including things like attempts at unauthorized access, or missing email addresses), we want to log the entire stack? That seems wrong.

**Review by delventhalz — CHANGES_REQUESTED** [review]

File: server/api/index.js (hunk 1 above)
Comment (RyanLassigBanks) [reply server/api/index.js:135]: Perhaps we should have different error handlers then? There are definitely times when you want the stack trace logged for debug purposes.

Comment (delventhalz) [reply server/api/index.js:135]: A quick fix to this would be logging the stack trace when the status is 500 or undefined.

**Comment by delventhalz** [comment]
Suggest closing this PR as the docker-compose fix has already gone in elsewhere, and there doesn't seem to be any further conversation on the stack trace logging.

**Pull request closed without merging**